commit adaa11c7588be3d6d78549ab60bbc451b10ae609
Author: Vitaly Kuznetsov <vkuznets@redhat.com>
Date:   Wed Dec 7 01:16:25 2016 -0800

    hv: allocate synic pages for all present CPUs
    
    commit 421b8f20d3c381b215f988b42428f56fc3b82405 upstream.
    
    It may happen that not all CPUs are online when we do hv_synic_alloc() and
    in case more CPUs come online later we may try accessing these allocated
    structures.
    
    Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1316_adaa11c7588be3d6d78549ab60bbc451b10ae609.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1316_adaa11c7588be3d6d78549ab60bbc451b10ae609.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
